contracts/CDPosition.sol

```diff
@@ -18,6 +19,8 @@ contract CDPosition {
         uint256 shares; // Total vault shares allocated to this position
     }
 
+    constructor(address admin) AccessController(admin) {}
```

Comment:
Should we set initialized flag on methods that dont have init method?

Reply:
I like the idea to put it on the constructor a lot but it's not valid apparently (no modifiers on constructor). I'm leaning towards putting `init` methods on everything. Most will probably need it and for others I don't see a downside.

Reply:
+1 on init method on everything

Reply:
Will address in the next PR